docs/01-app/03-api-reference/04-functions/setTagStale.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: setTagStale
+description: API Reference for the setTagStale function.
+---
+
+`setTagStale` allows you to mark a cache tag and any pages that use that tag as stale for background revalidation on the next request, instead of the blocking revalidation used by [`revalidateTag`](/docs/app/api-reference/functions/revalidateTag).
+
+## Usage
+
+`setTagStale` can be called in Server Functions and Route Handlers.
+
+`setTagStale` cannot be called in Client Components or Middleware, as it only works in server environments.
+
+> **Good to know**: Unlike [`revalidateTag`](/docs/app/api-reference/functions/revalidateTag) which performs blocking revalidation, `setTagStale` marks tags for background revalidation that occurs on the next request. This provides better performance by avoiding blocking operations during user requests.
+
+## Parameters
+
+```tsx
+setTagStale(tags: string | string[]): void;
+```
+
+- `tags`: A string or array of strings representing the cache tags associated with the data you want to mark as stale. Each tag must not exceed 256 characters and values are case-sensitive.
```

Comment:
maximum 256 characters per tag, but is there a maximum number of tags in the array?

cc @lubakravche

Reply:
We also have the `cacheTag` function, https://nextjs.org/docs/app/api-reference/functions/cacheTag, and the signature of that one is `cacheTag(...tags:string[])` - 

Should we align with that one? The example below would look a bit weird though 🤔 :

```ts
  const { tags } = await request.json()

  setTagStale(...tags)
```